parent/pom.xml

```diff
@@ -532,12 +532,12 @@ Bundle-Category: jackrabbit
             <dependency>
                 <groupId>org.apache.httpcomponents</groupId>
                 <artifactId>httpmime</artifactId>
-                <version>4.5.3</version>
+                <version>4.5.8</version>
```

Comment:
These should use the most recent version (4.5.13)

Reply:
Same as above